go.mod

```diff
@@ -28,25 +28,24 @@ require (
 	go.opentelemetry.io/otel/metric v0.31.0
 	go.opentelemetry.io/otel/sdk v1.10.0
 	go.opentelemetry.io/otel/trace v1.10.0
-	google.golang.org/grpc v1.49.0
```

Comment:
Need to bump metrics as well to the corresponding version: i.e go.opentelemetry.io/otel/metric I believe it's v0.35.0 https://github.com/open-telemetry/opentelemetry-go/releases/tag/v1.12.0

Reply:
Thanks! that was the hint that I needed.  I also had to bump up the `contrib` versions as well.